src/5.2.cup_of_join.py

```
@@ -0,0 +1,32 @@
+"""
+Week 5 Ex 2 - cup_of_join
+"""
+def cup_of_join(*lists, sep=None):

```

If sep is not provided should concatenate "-" between collections...
-4

---

I know that in the original excerice it says that if sep is not provided I should concatenate "-" but I changed it because the test script that you provide us does not expect it and it failed for me when I used "-" instead of None.

---

the mistake in the tests is ours, the grade will be corrected accordingly
+4